dangerzone/podman/machine.py

```diff
@@ -81,6 +81,9 @@ def init(
             logger.info(f"Podman machine '{self.name}' already exists.")
             return
 
+        if platform.system() == "Windows":
+            ret = container_utils.subprocess_run(["wsl", "--update"], check=False)
```

Comment:
We don't seem to use `ret` here. Also, why pass `check=False`?

Reply:
(Live feedback):
> We're passing `check=False` because it might fail (for instance when there are no network connection), and if that's the case it's okay to not enforce it.